content/md/first-steps/manage-families.md

```diff
@@ -1,112 +1,102 @@
 ---
 
-# List families
+:::info
+If you are not familiar with the concept of families, you can first read our [What is a family?](/articles/what-is-a-family.html) article.
+:::
 
-To see your families, edit them or create a new family go to `Settings/Families`.
+# Manage your families
 
-![List of families](../img/Settings_Families.png)
+## See all your families
 
-# Create a family
+To see your families, edit them or create a new family go to `Settings/Families`.       
 
-:::info
-You must be granted a specific permission to be able to create a family.
-:::
+![List of families](../img/Settings_Families.png)
+
+## How to create a family?
 
-To create a family:
+Families are used to gather attributes that are common to several products. The completeness of your products is also defined at the family level.      
+To create a new family, follow these steps: 
 1.  Go to `Settings/Families`
 1.  Click on `Create family`
 ![Create a family](../img/Settings_Families_Create.png)
 1.  The creation of a family popin is prompted
 1.  Enter a new family code (this code must be unique)
 1.  Click on `Save`
 
-The family is now created.
-
-Once a family is created, you can later edit those family information:
-*   The family’s labels for the locales enabled for the catalog
+Your new family is now created. The family's `Properties` tab is displayed.     
+You can define some additional properties for your family:      
+*   An `attribute used as label (required)`: the attribute that will be used as «Label» for the products belonging to this family. The Label will appear in the product grid ("Label") column and it will also be used in the product form's header.
+**Only text and identifier attributes can be used as a product's Label.**
+*   An `attribute used as the main picture`: the attribute that will be used as «the main picture» for the products belonging to this family in the product grid (column "image") and in the product form's header.
+You can select an `image` or  `assets collection` (since 2.1) attribute types. Attributes that localizable and/or scopable cannot be used as the main product picture.
+*   The family’s `Labels translations`: allow you to translate your family name into the locales enabled in the PIM. 
+    
+These information can be edited afterwards.     
+
+Once a family is created, you can configure your family:
 *   Add/remove attributes from the family
 *   Edit these attributes’ completeness
-*   To select the attribute to be used as «label» for the products in the product grid (column "label") and in their product form
-*   To select the attribute to be used as «the main picture» for the products in the product grid (column "image") and in their product form header
 
 :::info
-The attributes available to be used as «the main picture» are attributes not localizable and not scopable, with the types `image` or  `assets collection` (since 2.1).
+You must be granted a specific permission to be able to create a family.
```

Comment:
`role permission` + add a link to the `Build your user roles` article. (https://help.akeneo.com/articles/build-your-user-roles.html)